File: Assets/Scripts/Unused Scripts/SingleSpawner.js
#pragma strict

public var enemy : GameObject;
public var maxEnemies : int = 4;
public var numEnemies : int;

var rand : Random;
public var time : int;
var delay : int;

function Start () {

}

function FixedUpdate () {
	if(time >= delay)
	{
		if(numEnemies < maxEnemies)
		{
			Instantiate(enemy,transform.position,transform.rotation);
		}
		numEnemies ++;
		time = 0;
	}
	time++;
}

File: Assets/Scripts/Enemies/SingleSpawner.cs
using UnityEngine;
using System.Collections;

public class SingleSpawner : MonoBehaviour {

	public GameObject enemy;
	public int maxEnemies = 4;
	public int numEnemies;
	
	Random rand;
	public int time;
	int delay;

	// Use this for initialization
	void Start () {
	
	}
	
	// Update is called once per frame
	void FixedUpdate () {
		if(time >= delay)
		{
			if(numEnemies < maxEnemies)
			{
				Instantiate(enemy,transform.position,transform.rotation);
			}
			numEnemies ++;
			time = 0;
		}
		time++;
	}
}
